raiden/transfer/log.py

```diff
@@ -83,6 +84,12 @@ def __init__(self, database_path):
             'FOREIGN KEY(statechange_id) REFERENCES state_changes(id)'
             ')'
         )
+        cursor.execute(
+            'CREATE TABLE IF NOT EXISTS state_events ('
```

Comment:
I purposefully named it as `state_event` to show that this is events generated by the state machine. I would rather not give it the generic name `event` since that is associated with too many other things. Like blockchain events, greenlet events e.t.c.

I understand you rationale and I would even go so far as to argue we should change the name of the class to `StateEvent` for the same reason above at some point in the future.